source/dubregistry/dbcontroller.d

```diff
@@ -39,9 +39,13 @@ class DbController {
 		//
 
 		// create indices
-		m_packages.ensureIndex([tuple("name", 1)], IndexFlags.Unique);
-		m_packages.ensureIndex([tuple("stats.score", 1)]);
-		m_downloads.ensureIndex([tuple("package", 1), tuple("version", 1)]);
+        IndexOptions opt;
+        opt.unique = true;
```

Comment:
want to fix the mixed tabs and spaces here? otherwise LGTM

Reply:
Yes thx. Done.